### DISCUSSION TITLE
Unale to install sketchybar: cp: '/private/tmp/homebrew-unpack ...' -> '/private/tmp/sketchybar-20240323-25013 ...': No such file or directory (os error 2)

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Homebrew/discussions#5221] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: f6a4f42cc346694fac94fa1643bc2809d383b6f1
Last commit: 4 days ago
Core tap JSON: 23 Mar 20:14 UTC
Core cask tap JSON: 23 Mar 20:14 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.44.0 => /opt/homebrew/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.1.2-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
> brew doctor
Your system is ready to brew.
```

### Description of issue

When I try `brew install sketchybar` I get this:

```
==> Downloading https://formulae.brew.sh/api/formula.jws.json

==> Downloading https://formulae.brew.sh/api/cask.jws.json
#=#=#
==> Fetching felixkratz/formulae/sketchybar
==> Downloading https://github.com/FelixKratz/SketchyBar/archive/refs/tags/v2.21.0
Already downloaded: /Users/yousuf/Library/Caches/Homebrew/downloads/9a836e24e5b6c66582029337f2cd10c2213d3ee66f8130f15eb318894f47954c--SketchyBar-2.21.0.tar.gz
==> Installing sketchybar from felixkratz/formulae
cp: '/private/tmp/homebrew-unpack20240323-25013-alyles/SketchyBar-2.21.0/./LICENSE.md' -> '/private/tmp/sketchybar-20240323-25013-peoclx/SketchyBar-2.21.0/LICENSE.md': No such file or directory (os error 2)
Error: Failure while executing; `/usr/bin/env cp -pR /private/tmp/homebrew-unpack20240323-25013-alyles/SketchyBar-2.21.0/. /private/tmp/sketchybar-20240323-25013-peoclx/SketchyBar-2.21.0` exited with 1. Here's the output:
cp: '/private/tmp/homebrew-unpack20240323-25013-alyles/SketchyBar-2.21.0/./LICENSE.md' -> '/private/tmp/sketchybar-20240323-25013-peoclx/SketchyBar-2.21.0/LICENSE.md': No such file or directory (os error 2)
```

**Comment by daeho-ro** [comment]
It is from the custom tap `felixkratz/formulae` and so you have to contact the owner of the tap.
https://github.com/FelixKratz/homebrew-formulae/issues

**Reply by YousufSSyed** [reply]
Does this error have anything to do with Homebrew itself or is it specific to the tap / package? 

That error message sounds like a Homebrew related error (but I don't know anything about the inner workings of Homebrew).

**Reply by SMillerDev** [reply]
It does not, it's in the package

**Reply by YousufSSyed** [reply]
I have this issue for SketchyVim as well, @SMillerDev @daeho-ro I never worked with homebrew as a developer, how could I look at and debug the issue myself?

**Reply by SMillerDev** [reply]
Is the maintainer of the tap not interested in fixing it?

**Reply by YousufSSyed** [reply]
https://github.com/FelixKratz/homebrew-formulae/issues/12#issuecomment-2018720062
> Maybe someone else has an idea, but I have never seen someone else report that bug and I can not reproduce.

**Reply by gromgit** [reply]
I just did a `brew install felixkratz/formulae/sketchybar` with zero errors, so there's a good chance it's something specific to the OP's system.

@YousufSSyed, are you running an anti-virus or other desktop management suite like Jamf or Kandji? If so, turn it off and try your `brew install` again.

**Reply by YousufSSyed** [reply]
Can I uninstall homebrew without uninstall all my packages though?

**Reply by YousufSSyed** [reply, marked as answer]
I uninstalled Homebrew, restarted, and restarted it and both SketchyBar and SketchyVim installed succesfully.
